src/vcpkg/postbuildlint.cpp

```diff
@@ -888,6 +888,62 @@ namespace vcpkg::PostBuildLint
         return LintStatus::SUCCESS;
     }
 
+    static LintStatus check_no_absolute_paths_in(const Filesystem& fs, const Path& dir, Span<Path> absolute_paths)
+    {
+        static constexpr std::array<StringLiteral, 11> extensions = {
```

Comment:
```suggestion
        static constexpr StringLiteral extensions[] = {
```

no reason to use `std::array` here.